doc: resolve ambiguity in `pull-requests.md` (#28125)
nodejs/node: doc/guides/contributing/pull-requests.md

```txt
subsystem: explain the commit in one line

> The body of the commit message can be several paragraphs, explaining things in

Reviewer: ```suggestion
The body of the commit message can one or more paragraphs, explaining things in
```

> The body of the commit message can one or more paragraphs, explaining things in

Reviewer: ```suggestion
The body of the commit message should be one or more paragraphs, explaining things in
```

Reviewer 2: I saying `should`, it might indicate that committer has to write details
Some commit messages without details might be good enough on their own, should we replace this back to `can be`

Reviewer: I think "should be" (as opposed to "must be") conveys that there really *should* be a commit message body but that it is not absolutely required. Using "can be" makes it sound up to the committer, and while that may be strictly true, I think we should encourage providing a message body. So I prefer "should" over "can".

more detail, possibly giving some background about the issue being fixed, etc.

> Please word-wrap to keep columns shorter than about 72 characters or so.

Reviewer: ```suggestion
Please word-wrap to keep columns shorter than 72 characters.
```

> Please do proper word-wrap and keep columns shorter than about 72 characters

Reviewer: ```suggestion
Please word-wrap to keep columns shorter than about 72 characters.
```

> or so. That way, `git log` will show things nicely even when it is indented.

Reviewer: Remove?

Author: @Trott Sorry, but what exactly should I remove here?

Reviewer: The whole line? The previous line can end with a period and this line be removed. Just a suggestion. Feel free to ignore if you disagree with it.

Author: @Trott Done!
